service/version_bundles.go

```diff
@@ -3,29 +3,13 @@ package service
 import (
 	"github.com/giantswarm/versionbundle"
 
-	v20 "github.com/giantswarm/kvm-operator/service/controller/v20"
-	v21 "github.com/giantswarm/kvm-operator/service/controller/v21"
-	v22 "github.com/giantswarm/kvm-operator/service/controller/v22"
-	v23 "github.com/giantswarm/kvm-operator/service/controller/v23"
-	"github.com/giantswarm/kvm-operator/service/controller/v23patch1"
-	v24 "github.com/giantswarm/kvm-operator/service/controller/v24"
-	"github.com/giantswarm/kvm-operator/service/controller/v24patch1"
-	v25 "github.com/giantswarm/kvm-operator/service/controller/v25"
-	v26 "github.com/giantswarm/kvm-operator/service/controller/v26"
+	"github.com/giantswarm/kvm-operator/service/controller"
 )
 
 func NewVersionBundles() []versionbundle.Bundle {
```

Comment:
Since there's just one version bundle, we can get rid of this file and move the initialization into `service/service.go` https://github.com/giantswarm/kvm-operator/blob/master/service/service.go#L218